### ISSUE TITLE
App crash - [DSMPTNPDFViewController toolOverlayView]: unrecognized selector sent to instance 0x7f8c57710ec0

### ISSUE THREAD

**Issue by PradeshKumar** [docusign/native-ios-sdk#32]
Hi, when I'm trying to present the cached template in the offline mode, I'm facing an app crash.

-[DSMPTNPDFViewController toolOverlayView]: unrecognized selector sent to instance 0x7f8c57710ec0
2020-06-03 01:58:11.793362+0530 docusign-sdk-sample-swift[51401:1053548] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[DSMPTNPDFViewController toolOverlayView]: unrecognized selector sent to instance 0x7f8c57710ec0'

```
> *** First throw call stack:
> (
> 	0   CoreFoundation                      0x000000010e5018db __exceptionPreprocess + 331
> 	1   libobjc.A.dylib                     0x0000000109c7cac5 objc_exception_throw + 48
> 	2   CoreFoundation                      0x000000010e51fc94 -[NSObject(NSObject) doesNotRecognizeSelector:] + 132
> 	3   UIKitCore                           0x0000000119395235 -[UIResponder doesNotRecognizeSelector:] + 287
> 	4   CoreFoundation                      0x000000010e506623 ___forwarding___ + 1443
> 	5   CoreFoundation                      0x000000010e508418 _CF_forwarding_prep_0 + 120
> 	6   DocuSignSDK                         0x000000010a96e0c1 -[PTNToolManager newToolSetup] + 113
> 	7   DocuSignSDK                         0x000000010a96dcb2 -[PTNToolManager initWithPDFViewCtrl:] + 306
> 	8   DocuSignSDK                         0x000000010a86ea85 -[DSMPTNToolManager initWithPDFViewCtrl:toolManagerDelegate:annotationManager:] + 93
> 	9   DocuSignSDK                         0x000000010a8849dc -[DSMNativeSigningViewController initializeToolManagerForPDFViewController:] + 195
> 	10  DocuSignSDK                         0x000000010a883706 -[DSMNativeSigningViewController viewDidLoad] + 278
> 	11  UIKitCore                           0x0000000118d660f7 -[UIViewController loadViewIfRequired] + 1183
> 	12  UIKitCore                           0x0000000118d66524 -[UIViewController view] + 27
> 	13  UIKitCore                           0x0000000118cd7690 -[UINavigationController preferredContentSize] + 197
> 	14  UIKitCore                           0x0000000118c8930a -[UIPresentationController preferredContentSizeDidChangeForChildContentContainer:] + 64
> 	15  UIKitCore                           0x0000000118c8529f __56-[UIPresentationController runTransitionForCurrentState]_block_invoke + 114
> 	16  UIKitCore                           0x00000001193740e9 _runAfterCACommitDeferredBlocks + 318
> 	17  UIKitCore                           0x00000001193635b9 _cleanUpAfterCAFlushAndRunDeferredBlocks + 358
> 	18  UIKitCore                           0x0000000119390923 _afterCACommitHandler + 124
> 	19  CoreFoundation                      0x000000010e4682c7 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 23
> 	20  CoreFoundation                      0x000000010e46278e __CFRunLoopDoObservers + 430
> 	21  CoreFoundation                      0x000000010e462e01 __CFRunLoopRun + 1505
> 	22  CoreFoundation                      0x000000010e4624d2 CFRunLoopRunSpecific + 626
> 	23  GraphicsServices                    0x00000001137d72fe GSEventRunModal + 65
> 	24  UIKitCore                           0x0000000119368fc2 UIApplicationMain + 140
> 	25  docusign-sdk-sample-swift           0x000000010901c53b main + 75
> 	26  libdyld.dylib                       0x00000001105ba541 start + 1
> 	27  ???                                 0x0000000000000001 0x0 + 1
> )
> libc++abi.dylib: terminating with uncaught exception of type NSException
```

**Only, I'm facing this issue on the latest DocuSign SDK Version - 2.3**

**Comment by ashokds** [comment]
Hi @PradeshKumar ,

Please find a fix for this issue on the `v2.3.1` branch. You could add the following to resolve it on your end:

`pod 'DocuSign', :git => 'https://github.com/docusign/native-ios-sdk.git', :branch => "release/2.3.1"`

For pod related troubleshooting, please refer: https://github.com/docusign/native-ios-sdk/blob/master/support-files/Integration-Troubleshooting.md

Thanks~

**Comment by ashokds** [comment]
Fixed with `v2.3.1`
